opts/port.go

```diff
@@ -147,6 +148,9 @@ func ConvertPortToPortConfig(
 	ports := []swarm.PortConfig{}
 
 	for _, binding := range portBindings[port] {
+		if binding.HostIP != "" && binding.HostIP != "0.0.0.0" {
+			logrus.Warnf("ignoring IP-address (%s:%s:%s) service will listen on '0.0.0.0'", binding.HostIP, binding.HostPort, port)
```

Comment:
Should this be logrus? I think usually we just print directly to stderr? But also it's weird to write to stdio in this package.

Reply:
I think I used it based on other examples I found in the code-base (e.g. https://github.com/docker/cli/blob/cea4d37bcae4ae6f9d216ba1dda252227600c3b1/cli/compose/loader/loader.go#L475), but open to suggestions 👍